lib/session/call-session.js

```diff
@@ -2067,6 +2067,11 @@ Duration=${duration} `
       this.logger.error('CallSession:replaceEndpoint cannot be called without stable dlg');
       return;
     }
+    // When this call kicked out from conference, session need to replace endpoint
+    // but this.ms might be undefined/null at this case.
+    if (!this.ms) {
```

Comment:
can we just have
```
this.ms = this.ms || this.getMS();
```

Reply:
fixed, the this.getMS() already done that.

this.ms = this.getMS();